packages/placement-ordering/src/__tests__/ordering.test.js

```diff
@@ -98,6 +95,47 @@ describe('ordering', () => {
       });
     });
 
-  });
+    describe('choice -> target with removing tiles', () => {
```

Comment:
i feel that there is an opportunity to reduce the amount of boiler plate code in these tests... I'd like to see an api like: 

```javascript

describe('choice -> target - with removing tiles', () => {
   // where assertMoveFromModel is a function where all the boilerplate is set up, with all the config etc.
   const assertMove = assertMoveFromModel( initialState );

  assertMove( 3, 0, [ choice('0'), choice('1'), target(), target()]);
 //... more assert moves...
});

```